aleph/views/sessions_api.py

```diff
@@ -32,6 +37,18 @@ def _get_credential_authz(credential):
         return Authz.from_role(role=role)
 
 
+def _get_session_id(request):
```

Comment:
I feel this could be done in the `record_audit` function mentioned above -- it doesn't really have anything to do with Authz (although authz may influence it on the front-end, but that's more ... coincidence).